tensorboard/plugins/scalar/BUILD

```diff
@@ -15,6 +15,7 @@ py_library(
     srcs = ["scalars_plugin.py"],
     srcs_version = "PY2AND3",
     deps = [
+        ":metadta",
```

Comment:
typo: metadata

Reply:
That's embarrassing. I blame chrome remote desktop.